dpnp/dpnp_iface_arraycreation.py

```diff
@@ -730,6 +1049,31 @@ def empty(
 
     For full documentation refer to :obj:`numpy.empty`.
 
+    Parameters
+    ----------
+    shape : {int, sequence of ints}
+        Shape of the new array, e.g., (2, 3) or 2.
+    dtype : dtype, optional
+        The desired dtype for the array, e.g., dpnp.int32. Default is the default floating point
+        data type for the device where input array is allocated.
+    order : {"C", "F"}, optional
+        Memory layout of the newly output array. Default: "C".
+    device : {None, string, SyclDevice, SyclQueue}, optional
+        An array API concept of device where the output array is created.
+        The `device` can be ``None`` (the default), an OneAPI filter selector string,
+        an instance of :class:`dpctl.SyclDevice` corresponding to a non-partitioned SYCL device,
+        an instance of :class:`dpctl.SyclQueue`, or a `Device` object returned by
+        :obj:`dpnp.dpnp_array.dpnp_array.device` property.
+    usm_type : {"device", "shared", "host"}, optional
+        The type of SYCL USM allocation for the output array.
+    sycl_queue : {None, SyclQueue}, optional
+        A SYCL queue to use for output array allocation and copying.
+
+    Returns
+    -------
+    out : dpnp.ndarray
+        Array of uninitialized data of the given shape, dtype, and order.
+
     Limitations
     -----------
     Parameter `order` is supported only with values ``"C"`` and ``"F"``.
```

Comment:
Maybe it is cleaner to remove it (here and in similar places) now that we indicate supported ones in definition of `order`. what is your thought?